subjects/DPxAI/skeleton/README.md

```diff
@@ -0,0 +1,77 @@
+## Skeleton
+
+> Brainpower mode
+
+### Context
+
+To craft your robot friend, you first need to build its skeleton, and that's where HTML comes in. HTML is the blueprint that sets up the structure of your robot, giving it shape and form. With HTML, you'll create the framework that holds all your amazing ideas together. Ready to start building? Let’s dive in and create the foundation of your robot buddy!
+
+> For the whole quest, the principle is to iterate over your code: when you finish an exercise, copy-paste your code to use it in the next one.
+
+> Contrary to what you will see in the videos, you don't have to run a server on your machine to use the html css javascript thanks to the integrated code editor that takes care of all that.  
+> We've left the videos for the other educational content they provide.
+
+### Files naming
+
+- for the HTML files: `name-of-the-exercise.html`
+- for the CSS files: when you have to link one, it's named like so: `name-of-the-exercise.css`
+- for the JS files, when you have to link one, it's named like so: `name-of-the-exercise.js`
+
+### Resources
+
+We provide you with some content to get started smoothly, check it out!
+
+- Video [Where to start](https://www.youtube.com/watch?v=i2PWvTERU-g&list=PLDa5D3mQAy7SKRLbYIEAcPrbQxrR_g1uM&index=1)
+- Video [Basic set up of an HTML page](https://www.youtube.com/watch?v=QtKoO7tT-Gg&list=PLHyAJ_GrRtf979iZZ1N3qYMfsPj9PCCrF&index=1)
+- Video [Different HTML tags overview](https://www.youtube.com/watch?v=Al-Jzpib8VY&list=PLHyAJ_GrRtf979iZZ1N3qYMfsPj9PCCrF&index=2)
+
+Those videos are accompanying you step by step in each exercise, but if you want to check right away all the notions covered in the quest, you can watch the whole playlist [Web - HTML, CSS & DOM JS](https://www.youtube.com/playlist?list=PLHyAJ_GrRtf979iZZ1N3qYMfsPj9PCCrF).
+
+> Your working environment may not be exactly the same as what you see in the videos or documentation, just try to adapt your work according to your research and discoveries.  
+> Don't be afraid to try!
+
+### Instructions
```

Comment:
From the instructions I would expect the files to be delivered to be an `.html` file, while in the current ref is `skeleton.js`.

```json
# 20101.json
{
  "type": "exercise",
  "name": "skeleton",
  "attrs": {
    "displayedName": "Skeleton",
    "expectedFiles": ["skeleton.js"],
    "language": "dom",
    "validations": [
      {
        "type": "tester",
        "testImage": "ghcr.io/01-edu/test-dom",
        "cooldown": 0
      }
    ]
  }
}
```

Reply:
The refs will be recreated for the new piscine.